### PR TITLE
Update stuff for 0.15.0.

### PR DESCRIPTION
**Pull request by sean-mcmanus**
Update stuff for 0.15.0.

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 0.15.0: February 1, 2018
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: Feb 14? #Closed

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Maybe. I wasn't sure if we had other stuff we wanted to ship for February. I wanted to ship earlier so we could get usage data to see if crashes are still an issue or not, particularly with the new IntelliSense compiler. #Closed

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Changed it to the 5th. #Closed

**Review by bobbrow — APPROVED** [review]
:shipit:

**Review by grdowns — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
I'm not checking this in until the 0.15.0 binary is finished building/submitted (the binary won't have Griffin's change, but it'll have the other fixes) and I'll update the binary again later before the insider release.

**Comment by sean-mcmanus** [comment]
fwlinks point to the update binaries now.

**Pull request merged**